### DISCUSSION TITLE
CVE-2024-51744: Bad documentation of error handling in ParseWithClaims can lead to potentially dangerous situations in golang-jwt

### DISCUSSION THREAD

**Discussion by manfred-h** [helmfile/helmfile#1772] [Questions] [answered]
Are there any plans for [CVE-2024-51744](https://www.cve.org/CVERecord?id=CVE-2024-51744)?

**Comment by yxxhero** [comment, marked as answer]
@manfred-h helmfile don't depend golang-jwt directly.

**Reply by manfred-h** [reply]
@yxxhero thanks for your response! Indeed you're right, I checked both helmfile and vendor supplied/referenced sources myself and can confirm, `ParseWithClaims` is not used anywhere!

**Reply by yxxhero** [reply]
@manfred-h thanks for your issue. please ping me if you need more help.
